### PR TITLE
Tiny doc update to remote build trigger

### PR DESCRIPTION
**Pull request by treefern**
### Proposed changes
Tiny doc update (more comments in build trigger Action): explanation of triggering build in other doc repo

### Checklist (replace `[ ]` with `[x]` to check off)
- None are applicable

### CODE REVIEW

File: .github/workflows/trigger_docs.yaml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 
 name: Trigger DEA Docs Build
+#Documentation in this repo is not built in this GitHub Action. Instead it is built by a separate Action in the dea-docs repo.
+#See here: https://github.com/GeoscienceAustralia/dea-docs/blob/master/.github/workflows/main.yml
+
+#The below action triggers a build in dea-docs when changes are pushed to the 'stable' branch in the current repo
```
Comment (pindge) [inline .github/workflows/trigger_docs.yaml]: yaml style guide suggests a space following `#` https://developers.home-assistant.io/docs/documenting/yaml-style-guide/#comments

**Review by pindge — CHANGES_REQUESTED** [review]

File: .github/workflows/trigger_docs.yaml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 
 name: Trigger DEA Docs Build
+#Documentation in this repo is not built in this GitHub Action. Instead it is built by a separate Action in the dea-docs repo.
```
Comment (treefern) [inline .github/workflows/trigger_docs.yaml]:
```suggestion
# Documentation in this repo is not built in this GitHub Action. Instead, it is built by a separate Action in the dea-docs repo.
```

File: .github/workflows/trigger_docs.yaml [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+#See here: https://github.com/GeoscienceAustralia/dea-docs/blob/master/.github/workflows/main.yml
```
Comment (treefern) [inline .github/workflows/trigger_docs.yaml]:
```suggestion
# See here: https://github.com/GeoscienceAustralia/dea-docs/blob/master/.github/workflows/main.yml
```

File: .github/workflows/trigger_docs.yaml (hunk 1 above)
Comment (treefern) [inline .github/workflows/trigger_docs.yaml]:
```suggestion
# The below action triggers a build in dea-docs when changes are pushed to the 'stable' branch in the current repo
```

File: .github/workflows/trigger_docs.yaml [hunk 4]
```diff
@@ -15,4 +19,4 @@ jobs:
         with:
           token: ${{ secrets.REPO_PAT }}
           repository: GeoscienceAustralia/dea-docs
-          event-type: republish-docs
\ No newline at end of file
+          event-type: republish-docs
```
Comment (treefern) [inline .github/workflows/trigger_docs.yaml:22]:
```suggestion
          event-type: republish-docs

```

**Review by omad — APPROVED** [review]

**Pull request merged**